certbot/auth_handler.py

```diff
@@ -33,14 +33,18 @@ class AuthHandler(object):
         and values are :class:`acme.messages.AuthorizationResource`
     :ivar list achalls: DV challenges in the form of
```

Comment:
I'm wondering if it is still true cause `achalls` doesn't seems to be a parameter anymore.